backend/src/test/java/gov/cdc/usds/simplereport/service/PersonServiceTest.java

```diff
@@ -94,18 +98,130 @@ void roundTrip() {
         null,
         false,
         false);
+    _service.addPatient(
+        _site2.getInternalId(),
+        "BAZ",
+        "Chaz",
+        null,
+        "Pizzazz",
+        null,
+        LocalDate.of(1865, 12, 25),
+        _dataFactory.getAddress(),
+        "5555555555",
+        PersonRole.STAFF,
+        null,
+        null,
+        null,
+        null,
+        false,
+        false);
     List<Person> all =
         _service.getPatients(null, PATIENT_PAGEOFFSET, PATIENT_PAGESIZE, false, null);
-    assertEquals(2, all.size());
+    assertEquals(3, all.size());
+    // includes patients whose facility is null
+    List<Person> site1Patients =
+        _service.getPatients(
+            _site1.getInternalId(), PATIENT_PAGEOFFSET, PATIENT_PAGESIZE, false, null);
+    assertEquals(2, site1Patients.size());
+  }
+
+  @Test
+  @WithSimpleReportStandardUser
+  void addPatient_standardUser_successDependsOnFacilityAccess() {
```

Comment:
Better practice would be to have different test case for the positive and negative tests—either using a tailored `@WithMockUser` or with custom annotations for the interesting cases.

Reply:
wouldn't be able to use `@WithMockUser` since that requires a static declaration of authorities when I can't provide them (since the authorities need DB-generated facility UUIDs). otherwise noted.